onnxruntime/test/providers/provider_test_utils.h

```diff
@@ -317,8 +317,15 @@ class OpTester {
                                optional<float>(), optional<float>()));
   }
 
+  /*
+  * Use this API to add an input *edge* to the node/op being tested that won't 
+  * have any data passed into.
+  * Such an edge will have the qualifier OpSchema::Optional in the schema.
+  * This is exposed to ensure the op kernel implementations can be tested to handle 
+  * presence/absence of such "data-less" optional edges.
+  */
   template <typename T>
-  void AddMissingOptionalInput() {
+  void AddDataLessInput() {
```

Comment:
Should we call this AddOptionalInputEdge() and the other one AddOptionalOutputEdge?

Reply:
Done